crates/bevy_app/src/app.rs

```diff
@@ -257,6 +263,35 @@ impl App {
             sub_app.run();
         }
 
+        // move schedules out of the world so we can modify it as we iterate
+        // through the schedule events
+        let mut schedules = self
+            .world
+            .remove_resource::<Schedules>()
+            .expect("No Schedules resource in world");
+
+        if let Some(mut sched_events) = self.world.get_resource_mut::<Events<ScheduleEvent>>() {
```

Comment:
Nit: please just use whole words (like schedule) if at all possible: it's much easier to read, especially for people with weaker English skills.

Reply:
I was curious why bevy used full names internally and in all examples; will do.